test/test_operators.py

```diff
@@ -236,10 +273,184 @@ def sym_eval(expr, x_vec):
             max(flatten(
                 componentwise_norms(dcoll, test_grad - exact_grad, np.inf),
                 actx) / err_scale)
+        print(f"{actx.to_numpy(h_max)=}\n{actx.to_numpy(grad_err)=}")
 
         eoc.add_data_point(actx.to_numpy(h_max), actx.to_numpy(grad_err))
 
     assert (
         eoc.order_estimate() >= order - 0.5
         or eoc.max_error() < tol
     )
+
+    if do_rotation_test:
```

Comment:
Could this and the wonk test possibly be implemented by adding additional parameters to `test_grad_operator` instead of duplicating the test loop logic? Maybe something along the lines of:

```python
@pytest.mark.parametrize(("dim", "mesh_config"), [
    (1, "box"),
    (2, "box"),
    (3, "box"),
    (2, ("rotated box", np.array([0, 0, 1]))),
    (3, ("rotated box", np.array([0, 0, 1]))),
    (3, ("rotated box", np.array([0, 1, 1]))),
    (3, ("rotated box", np.array([1, 1, 1]))),
    (2, "wonky box"),
    (3, "wonky box"),
])
```
?

Reply:
Thanks, loads better.  See what you think of this ([bd0d0fb](https://github.com/illinois-ceesd/mirgecom/pull/1043/commits/bd0d0fba275b07b503493b061f3d1defed8c51eb).  The actual wonk tests are not currently done because even tets are failing those.  I prefer to leave it in-place so I can tinker with it more.